src/components/DefineCompany/YourCompany.vue

```diff
@@ -8,52 +8,65 @@
     <div class="section-container px-8">
       <!--TODO: Replace container content with Name Request Summary when it is ready -->
       <v-layout row>
-        <v-flex md4>
-          <label><strong>Company Name</strong></label>
+        <v-flex md3>
+          <v-layout column>
```

Comment:
This may not be the time to figure out the layout but I'm bringing it up anyway to think about.

First, is this code really needed for a single column?

Second, I think all rows in this section should be in a single v-layout block (so that all columns are the same width).

Reply:
This specific column is for keeping the label and v-chip in alignment.

I definitely agree though, this layout could certainly use another look, especially as we continue on with adding more correction options to this `yourCompany` component.

Reply:
OK, we've thought about it... and we can revisit later.